src/api/status/src/js/octokit.js

```diff
@@ -0,0 +1,7 @@
+const { Octokit } = require('@octokit/core');
+const { retry } = require('@octokit/plugin-retry');
+
+const MyOctokit = Octokit.plugin(retry);
+const octokit = new MyOctokit();
```

Comment:
We should probably support `auth` via `env` so that we can pass in an access token for rate limiting later:

- https://github.com/octokit/octokit.js#constructor-options
- https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token
- https://github.com/octokit/octokit.js#authentication

If it's not set, we won't use one (e.g., it's optional in dev/tests), but when present (staging/prod) we'd add it.